content/knowledge_base/consensus/avalanche/concepts.md

```diff
@@ -0,0 +1,35 @@
+# Consensus algorithm concepts behind Avalanche
```

Comment:
```suggestion
# Avalanche Consensus Algorithm Concepts
```

WDYT of changing it to this? For the capitalization of titles in our markdown documents, please follow the first point in the [Markdown style guidelines](https://github.com/optakt/optakt.github.io/blob/master/content/style/markdown.md#general).

Reply:
This has been fixed https://github.com/optakt/optakt.github.io/pull/13/commits/211354094b1da3445d32b93c04cf950b6a935394#diff-7b8cdfdde84c7352f51f3cc642d28c90f9d1283870319c81956812353289fb32R1-R2